### PR TITLE
feat: Nethermind support

### PR DESCRIPTION
**Pull request by Wolmin**
DEV-11425
DEV-11426
This PR provides nethermind docker-compose files, as well as an update to already existing clients

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-11425 Nethermind - docker repos](https://app.clickup.com/t/2645698/DEV-11425)
Task linked: [DEV-11426 Bump client versions](https://app.clickup.com/t/2645698/DEV-11426)

File: docker-compose.nethermind-prysm.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      --bootstrap-node $CONSENSUS_BOOTSTRAP_NODE_1,$CONSENSUS_BOOTSTRAP_NODE_2
+      --execution-endpoint=http://localhost:8551
+      --jwt-secret=/configs/jwt.hex
+      --suggested-fee-recipient $FEE_RECIPIENT
+      --min-sync-peers=0
+      --monitoring-host 0.0.0.0
+      --grpc-gateway-host 0.0.0.0
+      --rpc-host 0.0.0.0
+      --verbosity $PRYSM_VERBOSITY
+      --p2p-max-peers 250
```
Comment (mxmar) [inline docker-compose.nethermind-prysm.yml:61]: please decrease max peers number to the defaults

File: docker-compose.nethermind-prysm.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    command: >
+      --accept-terms-of-use
+      --genesis-state=/configs/genesis.ssz
+      --chain-config-file=/configs/config.yaml
+      --datadir=/consensus_data
+      --bootstrap-node $CONSENSUS_BOOTSTRAP_NODE_1,$CONSENSUS_BOOTSTRAP_NODE_2
+      --execution-endpoint=http://localhost:8551
+      --jwt-secret=/configs/jwt.hex
+      --suggested-fee-recipient $FEE_RECIPIENT
+      --min-sync-peers=0
```
Comment (mxmar) [inline docker-compose.nethermind-prysm.yml:56]: Please change to 1

File: docker-compose.nethermind-prysm.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      --monitoring-host 0.0.0.0
+      --grpc-gateway-host 0.0.0.0
+      --rpc-host 0.0.0.0
+      --verbosity $PRYSM_VERBOSITY
+      --p2p-max-peers 70
+      --subscribe-all-subnets
+      --minimum-peers-per-subnet 0
```
Comment (mxmar) [inline docker-compose.nethermind-prysm.yml:63]: Please change to 1

File: docker-compose.nethermind-prysm.yml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      --contract-deployment-block 0
+      --block-batch-limit 512
+      --block-batch-limit-burst-factor 10
+      --p2p-host-ip $EXTERNAL_IP
+      --checkpoint-sync-url=$CHECKPOINT_SYNC_URL
```
Comment (mxmar) [inline docker-compose.nethermind-prysm.yml:68]: We don't need `--genesis-beacon-api-url` in Prysm v5+?

**Review by mxmar — CHANGES_REQUESTED** [review]

File: docker-compose.nethermind-prysm.yml (hunk 4 above)
Comment (Wolmin) [reply docker-compose.nethermind-prysm.yml:68]: We don't - what would pair with checkpoint flag is `--weak-subjectivity-checkpoint`, but this is only to ensure the additional safety measures to limit trust, plus it needs to be in format of current block root + epoch, so I think that as long as we are using our own checkpoint solution (checkpointz) then leaving simply this flag is fine

Comment (mxmar) [reply docker-compose.nethermind-prysm.yml:68]: Makes sense to me. As long as `genesis-beacon-api-url` is not required for checkpointz sync in Prysm v5+

**Review by mxmar — APPROVED** [review]

**Pull request merged**